webapp/javascript/components/ExportData.tsx

```diff
@@ -64,6 +65,21 @@ function ExportData(props: ExportDataProps) {
     downloadAnchorNode.remove();
   };
 
+  const exportPProf = function (flamebearer: FlamebearerProfile) {
+    const url = `${buildRenderURL({
+      from: flamebearer.metadata.startTime,
+      until: flamebearer.metadata.endTime,
+      query: flamebearer.metadata.query,
+      maxNodes: flamebearer.metadata.maxNodes,
+    })}&format=pprof`;
+
+    const downloadAnchorNode = document.createElement('a');
+    downloadAnchorNode.setAttribute('href', url);
+    document.body.appendChild(downloadAnchorNode); // required for firefox
```

Comment:
Just curious, instead of doing this with javascript wouldn't it be simpler to use an anchor tag in the export menu?

Reply:
Hey!

That's a really good question, there are a few reasons (one stronger, the others meh):

1 - We are already doing that for png/JSON: 
https://github.com/pyroscope-io/pyroscope/blob/3435a212aa659d5240f84702926833d351a9d44f/webapp/javascript/components/ExportData.tsx#L53-L64
https://github.com/pyroscope-io/pyroscope/blob/3435a212aa659d5240f84702926833d351a9d44f/webapp/javascript/components/ExportData.tsx#L27-L44

By maintaining consistency it becomes easier to a) understand the code b) refactor c) test

2 - There's an argument to be made that the anchor tag being used for navigating to another page and button for actions. Here we aren't really navigating to anywhere.

3 - The operation is delayed and obscured, which currently does not seem to matter >much<, but it will for cases like grafana where you need bit more involvement than simply pointing to a URL.

But honestly, both ways work. I woulnd't care too much about this.